docs/machine-learning/resources/glossary.md

```diff
@@ -91,32 +91,32 @@ In [classification](#classification), the recall for a class is the number of it
 
 ## Regression
 
-A supervised machine learning task where the output is a real value, for example, double. Examples include forecasting and predicting stock prices.
+A [supervised machine learning](#supervised-learning) task where the output is a real value, for example, double. Examples include predicting stock prices.
 
 ## Relative absolute error
 
-An evaluation metric that represents the error as a percentage of the true value.
+In [regression](#regression), an evaluation metric that is the sum of all absolute errors divided by the sum of distances between correct [label](#label) values and the average of all correct label values.
 
```

Comment:
I think for these evaluation metrics long term we should probably put in equations, as trying to parse through the wording (although accurate) can get confusing.

Reply:
@aditidugar that's true. I've also thought about inserting the link to related ML.NET API, so folks know immediately what to use from the library for an evaluation metric. And now, API remarks already contain equations (though not in pretty LaTeX format, but fine)

Reply:
Though, I haven't found API for the relative absolute error, but other evaluation metrics (RMS, RSquared, recall, etc) are supported by the library.

Reply:
@pkulikov yes, I think that's a good idea! Although, maybe I am not looking in the right place, but I don't seem to see a lot of the equations laid out in the API reference either (for example, https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.models.classificationmetrics.accuracymacro?view=ml-dotnet). 

Either location (API ref or glossary itself) seems fine for the equation, but we should make sure all of them are there.

Reply:
OK, then I'll make another PR with API links.
As for the equations, we'll find later a good place for them.